src/theme.rs

```diff
@@ -525,13 +525,30 @@ pub trait Theme {
             })
             .collect::<String>();
 
-        let footer = bar_color
-            .apply_to(format!(
-                "{S_CONNECT_LEFT}{horizontal_bar}{S_CORNER_BOTTOM_RIGHT}\n{bar}\n",
-                horizontal_bar = S_BAR_H.to_string().repeat(width + 3),
-                bar = bar_color.apply_to(S_BAR),
-            ))
-            .to_string();
+        let footer = if is_outro {
```

Comment:
Additionally, I would use a "corner" symbol for outro block

```
│  
├  This is an outro note ───╮
│                           │
│  Some explanation         │
└───────────────────────────╯
```

Reply:
Are you referring to `├` instead of `◇`?

Reply:
Yes, because all outros are like not items, but rather footers that are visually signaling "this is the end".

Reply:
Fixed in latest push.